vuepress/docs/next/tutorials/create/pb/export-bundle-from-application.md

```diff
@@ -4,30 +4,36 @@ sidebarDepth: 2
 # Export and Publish a Bundle
 
 ## Overview
-Use the Entando bundler command to export a bundle of Entando components from an existing Entando application. An Entando bundle can be used to do the initial install of Entando components into an Entando application, migrate Entando components from one environment to another (e.g. Dev to QA), to provide a template for building a new Entando application, or as the skeleton of an Entando solution. The output of this command is the same bundle directory structure created by an Entando project including a bundle descriptor file.
+Use the bundler command to export a bundle of Entando components from an existing Entando Application. An Entando Bundle can be used 
```

Comment:
take out 'Entando' before components >> Use the bundler command to export a bundle of components from an existing Entando Application.

too many Entandos but not a big deal. I think in this case, since the components are from an Ent Application, it can be implied that they are Entando components

Reply:
three was too many... and two still is!